src/garage/envs/base.py

```diff
@@ -94,36 +92,36 @@ def _close_viewer_window(self):
                     glfw.destroy_window(self.env.viewer.window)
             elif any(package in self.env.spec._entry_point
                      for package in KNOWN_GYM_NOT_CLOSE_VIEWER):
+                from gym.envs.classic_control.rendering import (
+                    SimpleImageViewer)
+                from gym.envs.classic_control.rendering import Viewer
```

Comment:
I think we should delay these imports further, until we're sure that `self.env` has a viewer field. In other words, please split the if statement below into two nested statements, and put the imports in between them.